docker-compose.override.yml

```diff
@@ -5,6 +5,7 @@ x-airflow-env:
     - CIVICRM_API_KEY_FILE_PATH=/home/airflow/secrets/civi_api_key
     - CIVICRM_SITE_KEY_FILE_PATH=/home/airflow/secrets/civi_site_key
     - PATH_TO_TOGGL_API_TOKEN_FILE=/home/airflow/secrets/toggl.txt
+    - GOOGLE_APPLICATION_CREDENTIALS=/home/airflow/secrets/gmail_credentials.json
```

Comment:
Just a note: This will override the "default credentials", that are mounted below from `~/.config/gcloud/application_default_credentials.json` to `/tmp/credentials.json`, and then copied to `/home/airflow/.config/gcloud/` via the command defined in `docker-compose.yml` file.
i.e. in development I guess all GCP access will only work with the gmail service account credentials file.